components/CloudSignup/CloudSignup.tsx

```diff
@@ -3,13 +3,20 @@ import styled from "styled-components";
 import { all, css, StyledSystemProps } from "components/system";
 import { BoxProps } from "components/Box";
 import { renderSignupRequestForm } from "utils/cloud-signup";
+import { GTMEvent } from "utils/gtm";
 
 export default function CloudSignup(props: BoxProps) {
   const ref = useRef<HTMLDivElement | null>(null);
 
   useEffect(() => {
+    const API_URL = process.env.NEXT_PUBLIC_TELEPORT_API_URL;
+
     if (ref) {
-      renderSignupRequestForm(ref.current);
+      renderSignupRequestForm(ref.current, {
+        apiUrl: API_URL,
+        onSuccess: () =>
+          GTMEvent("formSubmit", { formType: "Teleport Pro Signup" }),
```

Comment:
You maaay want to add braces around the call just to be safe. It will not break anything right now, but GTMEvent return `Promise` (so you can optionally wait until event is sent and callback is fired), and while returning promise here is working now, who knows how GTM code will change in the future.

```suggestion
          { GTMEvent("formSubmit", { formType: "Teleport Pro Signup" }); },
```